sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/api/Azure.ResourceManager.RecoveryServicesSiteRecovery.netstandard2.0.cs

```diff
@@ -5090,161 +4199,1198 @@ public partial class ResourceHealthSummary
+        public SiteRecoveryVaultSettingCreateOrUpdateContent(Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.VaultSettingCreationProperties properties) { }
+        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.VaultSettingCreationProperties Properties { get { throw null; } }
+    }
+    public partial class SiteRecoveryVaultSettingProperties
+    {
+        internal SiteRecoveryVaultSettingProperties() { }
+        public Azure.Core.ResourceIdentifier MigrationSolutionId { get { throw null; } }
+        public string VMwareToAzureProviderType { get { throw null; } }
+    }
+    public partial class SiteRecoveryVCenterCreateOrUpdateContent
+    {
+        public SiteRecoveryVCenterCreateOrUpdateContent() { }
+        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.SiteRecoveryAddVCenterProperties Properties { get { throw null; } set { } }
+    }
+    public partial class SiteRecoveryVCenterPatch
+    {
+        public SiteRecoveryVCenterPatch() { }
+        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.SiteRecoveryUpdateVCenterProperties Properties { get { throw null; } set { } }
+    }
+    public partial class SiteRecoveryVCenterProperties
+    {
+        internal SiteRecoveryVCenterProperties() { }
+        public string DiscoveryStatus { get { throw null; } }
+        public string FabricArmResourceName { get { throw null; } }
+        public string FriendlyName { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.SiteRecoveryHealthError> HealthErrors { get { throw null; } }
+        public string InfrastructureId { get { throw null; } }
+        public string InternalId { get { throw null; } }
+        public System.Net.IPAddress IPAddress { get { throw null; } }
+        public System.DateTimeOffset? LastHeartbeatReceivedOn { get { throw null; } }
+        public string Port { get { throw null; } }
+        public string ProcessServerId { get { throw null; } }
+        public string RunAsAccountId { get { throw null; } }
+    }
+    public partial class SiteRecoveryVersionDetails
+    {
+        internal SiteRecoveryVersionDetails() { }
+        public System.DateTimeOffset? ExpireOn { get { throw null; } }
+        public Azure.ResourceManager.RecoveryServicesSiteRecovery.Models.SiteRecoveryAgentVersionStatus? Status { get { throw null; } }
+        public string Version { get { throw null; } }
+    }
+    public partial class SiteRecoveryVmDiskDetails
+    {
+        internal SiteRecoveryVmDiskDetails() { }
+        public string CustomTargetDiskName { get { throw null; } }
+        public Azure.Core.ResourceIdentifier DiskEncryptionSetId { get { throw null; } }
+        public string DiskId { get { throw null; } }
+        public string LunId { get { throw null; } }
+        public string MaxSizeMB { get { throw null; } }
+        public string TargetDiskLocation { get { throw null; } }
```

Comment:
Is this AzureLocation?

Reply:
Unable to confirm if it's `AzureLocation` or not, leave it as is.